documentation/database/database.md

```diff
@@ -0,0 +1,146 @@
+# Database Objects
```

Comment:
Add [here](https://github.com/nhsconnect/integration-adaptor-gp2gp/blob/main/OPERATING.md#database-design) a link to your file